### ISSUE TITLE
execute multiple commands

### ISSUE THREAD

**Issue by sharukhkhanajm** [Genymobile/scrcpy#1138]
hi I want  to know that how can I execute multiple commands like
adb devices
scrcpy --no-control
scrcpy --bit-rate 6M
scrcpy --max-fps 40
scrcpy --max-size 1028

in given commands, it stops at bit rate and max fps and max size will not be executed please help me.
thanks. :)

**Comment by npes87184** [comment]
Try `scrcpy --no-control --bit-rate 6M --max-fps 40 --max-size 1028`?

**Comment by sharukhkhanajm** [comment]
this is my .bat file
cd c:\Mirror
adb devices
scrcpy --bit-rate 6M --max-size 1028

and when I run this file I get this output

C:\Mirror>scrcpy --bit-rate 6M --max-size 1028
scrcpy-server.jar: 1 file pushed. 0.6 MB/s (19342 bytes in 0.031s)
INFO: Initial texture: 496x1024

I think the last command is running and the bit rate is being ignored what do you think ?

**Comment by rom1v** [comment]
No, it's not ignored. Try with `--bit-rate 1M`, you'll notice the difference in quality.
